spec.bs

```diff
@@ -10166,43 +10159,43 @@ The [=k-anonymity overrides=] are checked before and take precedence over the
     </table>
 </figure>
 
-The [=remote end steps=] are:
+<div algorithm="remote end steps">
+The <dfn export for="Protected Audience">remote end steps</dfn> are:
+
 1. If |parameters| is not a JSON [[ECMASCRIPT#sec-json-object|Object]], return a
-     [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+     [=error|WebDriver error=] with [=error code=] [=invalid argument=].
 1. Let |owner| be the result of [=getting a property=] named `"owner"` from
     |parameters|.
-1. If |owner| is [=undefined=] or is not a [=string=], return a
-   [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. If |owner| is {{undefined}} or is not a [=string=], return a

```

Comment:
this {{undefined}} links to the IDL one, but the webdriver2 one links to the ECMA script one https://www.ecma-international.org/ecma-262/5.1/#sec-4.3.9. I don't think that's a big deal, but probably easist to revert to `text: undefined; url: dfn-undefined` you had.

Reply:
Done